models/DataObject/ClassDefinition.php

```diff
@@ -315,7 +315,7 @@ public static function cleanupForExport(&$data)
             }
         }
 
-        if (method_exists($data, 'getChildren')) {
+        if (is_object($data) && method_exists($data, 'getChildren')) {
```

Comment:
how about early return at line 301?
```php
        if (!is_object($data)) {
            return;
        }
```

Reply:
might as well work. should I change the PR?

Reply:
yes, please!